pkgs/tools/security/mokutil/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, lib, fetchFromGitHub, autoreconfHook, pkg-config, openssl, efivar, keyutils }:
+stdenv.mkDerivation rec {
+  pname = "mokutil";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "lcp";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-dt41TCr6RkmE9H+NN8LWv3ogGsK38JtLjVN/b2mbGJs=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  buildInputs = [ openssl efivar keyutils ];
+
+  meta = {
+    homepage = "https://github.com/lcp/mokutil";
+    description = "The utility to manipulate machines owner keys which managed in shim";
+    license = lib.licenses.gpl3Only;
```

```suggestion
    license = lib.licenses.gpl3Plus;
```
The source code says GPL v3 or any later version. https://github.com/lcp/mokutil/blob/master/src/keyring.c